shared-libs/transitions/src/transitions/create_user_for_contacts.js

```diff
@@ -0,0 +1,127 @@
+const config = require('../config');
```

Comment:
Just noticed that all the other tests in this folder are dash separated, while this one is underscore separated. Would you mind renaming it and use dash instead of underscore? Thanks! 
![image](https://user-images.githubusercontent.com/35681649/193979687-0513e665-5f06-46c7-a063-119275bdf24f.png)